completion.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var completionCmd = &cobra.Command{
+	Use:       "completion [shell]",
+	Args:      cobra.OnlyValidArgs,
+	ValidArgs: []string{"bash", "zsh"},
+	Short:     "Generate shell completions",
+	Long: `To load completion run
+
+. <(jk completion)
+
+To configure your bash shell to load completions for each session add to your bashrc
+
+# ~/.bashrc or ~/.profile
+. <(jk completion)
```

Comment:
Could you add the instructions for `zsh` in the help text?

Reply:
Resolved in 7f17f06388c3ec0b2ee9972ce9830fd6785d6e69